.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,28 +4,35 @@ about: Create a report to help us improve
 title: "[BUG]"
 labels: bug
 assignees: ''
-
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+## Describe the bug
+
+<!-- A clear and concise description of what the bug is. -->
+
+## Steps to reproduce
 
-**To Reproduce**
-Steps to reproduce the behavior:
+<!-- 
```

Comment:
[imo] I prefer these uncommented so it's visible in markdown preview. WDYT?

Reply:
I would like to go with comments, because when I am submitting bug reports, its bothering to me that deleting lines that I don't need. Reproduction steps will have variety by reporters.